test/support/command_case.ex

```diff
@@ -10,17 +10,28 @@ defmodule Cog.CommandCase do
     quote location: :keep do
       import unquote(__MODULE__)
 
-      @command_module Keyword.fetch!(unquote(opts), :command_module)
-      @command_name Module.split(@command_module) |> List.last |> String.downcase
+      @command_module Keyword.get(unquote(opts), :command_module)
+      @command_tag Keyword.get(unquote(opts), :command_tag)
 
-      @moduletag commands: @command_name
+      @command_name (if @command_module do
```

Comment:
From reading the code it's not clear to me when or why each of these options should be used. An explanatory comment here would help greatly.